airflow/www/app.py

```diff
@@ -971,6 +971,10 @@ def action(self):
             # Flagging tasks as successful
             session = settings.Session()
             task_ids = [task_id]
+            end_date = ((dag.latest_execution_date or datetime.now())
+                        if future else execution_date)
+            start_date = dag.default_args['start_date'] if past else execution_date
```

Comment:
I can copy https://github.com/airbnb/airflow/blob/master/airflow/www/app.py#L1050-L1051?

Is there a possibility both of those are null? (If so I can default to execution date.